src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import ToggleSwitch from 'toggle-switch-react-native';
-import {Image, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
+import {Image, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import ToggleSwitch from '../../../components/ToggleSwitch';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ToggleSwitch` has been changed from an external library to a local file. Ensure that the new `ToggleSwitch` component has all the necessary functionalities and behaves as expected in different scenarios.
```diff
- import ToggleSwitch from 'toggle-switch-react-native';
+ import ToggleSwitch from '../../../components/ToggleSwitch';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->